pages/pricing/_pricing-add-ons.html

```diff
@@ -0,0 +1,115 @@
+      data-toggle="tooltip"
+      >Reference Architecture</label
+    >
+    <div class="text-center grid-item-center">
+      <div class="pricing-checkbox-wrapper">
+        <input
+          type="checkbox"
+          id="add-on-reference-architecture-standard"
+          {% unless include.already_included %}name="{{site.products.RefArch.key}}"{% endunless %}
+          class="addon-checkbox"
+          data-product-id="{{site.products.RefArch.stripeProductId}}"
+        >
+        <span class="light">+$<strong>4,950</strong> one-time</span>
+      </div>
+    </div>
+    <div class="text-center grid-item-center">
+      <div class="pricing-checkbox-wrapper pricing-checkbox-wrapper-auto">
+        <input
+          type="checkbox"
+          id="add-on-reference-architecture-enterprise"
+          {% unless include.already_included %}name="{{site.products.RefArch.key}}"{% endunless %}
+          disabled="disabled" 
+          checked="checked"
+          class="addon-checkbox"
+          data-product-id="{{site.products.RefArch.stripeProductId}}"
+        >
+        <span class="light">included</span>
+      </div>
+    </div>
+  </li>
+  <li class="grid-container grid-vertical-center">
+    <label
+      for="add-on-pro-support"
+      class="light"
+      title="Get support via email or a private Slack channel, with an SLA of 2 business days."
+      data-toggle="tooltip"
+      >Professional Support</label
+    >
+    <div class="text-center grid-item-center">
+      <div class="pricing-checkbox-wrapper">
+        <input
+          type="checkbox"
+          id="add-on-pro-support-standard"
+          {% unless include.already_included %}name="{{site.products.ProSupport.key}}"{% endunless %}
+          class="addon-checkbox"
+          data-product-id="{{site.products.ProSupport.stripeProductId}}"
+        >
+        <span class="light">+$<strong>1,950</strong> / month</span>
+      </div>
+    </div>
+    <div class="text-center grid-item-center">
+      <div class="pricing-checkbox-wrapper pricing-checkbox-wrapper-auto">
+        <input
+          type="checkbox"
+          id="add-on-pro-support-enterprise"
+          {% unless include.already_included %}name="{{site.products.ProSupport.key}}"{% endunless %}
+          disabled="disabled" 
+          checked="checked"
+          class="addon-checkbox"
+          data-product-id="{{site.products.ProSupport.stripeProductId}}"
+        >
+        <span class="light">included</span>
+      </div>
+    </div>
+  </li>
+  <li class="grid-container grid-vertical-center">
+    <label
+      class="light"
+      for="add-on-compliance"
+      title="Get access to our CIS Service Catalog, where all the modules are compliant with the CIS AWS Foundations Benchmark. If you pick this and the Reference Architecture add-on, the entire Reference Architecture will be compliant with the CIS AWS Foundations Benchmark!"
```

Comment:
Since the pricing page no longer has underlines on each row label, it's not obvious you can mouse-over them to see these tool tips. 

Would it make sense to add an effect where if you mouse over anywhere in the row (not just the text on the left, but anywhere), the entire row is slightly highlighted, and the tool tip shows up?

Reply:
We have added borders and changed the text color, also the tooltip is visible when you hover on the row.
For most of them, you can add tooltip content in pricing-table.yml. For the rows adds-on and pricing you can edit the content for the tooltip in files _pricing-add-ons and _pricing.

Reply:
Sorry, on the [preview page](https://deploy-preview-804--keen-clarke-470db9.netlify.app/pricing ), I can see the tooltip when I mouse over the text or near it, but I'm not seeing any hover effect (e.g., you mentioned "added borders"):

![Screen Shot 2023-07-05 at 11 21 09 AM](https://github.com/gruntwork-io/gruntwork-io.github.io/assets/711908/5da3c1ea-2ecc-4a96-9672-aa270afbeab3)

And if I mouse over the right two columns, I don't see the tooltip either:

![Screen Shot 2023-07-05 at 11 21 13 AM](https://github.com/gruntwork-io/gruntwork-io.github.io/assets/711908/68c5b7af-1313-4392-ae61-b55f24919915)

(Doh. Just realized the Mac screenshot tool hides the mouse pointer... Well, in the first screenshot, the mouse pointer is over the "CIS Compliance" text, and in the second screenshot, it's to the right of "$1,250/month" and left of "included.")

I'm using Chrome on macOS, if that helps.

Reply:
Ah, sorry, it was more like the comment saved for later. I did the changes but now they are being reviewed by our team, as soon as it is checked I will commit changes here.